pkgs/build-support/setup-hooks/wrap-gapps-hook/default.nix

```diff
@@ -17,24 +16,15 @@ makeSetupHook {
   propagatedBuildInputs = [
     # We use the wrapProgram function.
     makeWrapper
-  ] ++ lib.optionals isGraphical [
-    # TODO: remove this, packages should depend on GTK explicitly.
-    gtk3
-
-    librsvg
   ];
 
   # depsTargetTargetPropagated will essentially be buildInputs when wrapGAppsHook is placed into nativeBuildInputs
-  # the librsvg and gtk3 above should be removed but kept to not break anything that implicitly depended on its binaries
```

Comment:
The librsvg removal refers to being moved to `depsTargetTargetPropagated`. It still might be needed for rendering SVG icons with gdk-pixbuf.